packages/wallets/src/features/cashier/modules/WithdrawalVerification/__tests__/WithdrawalVerification.spec.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { useCountdown } from 'usehooks-ts';
+import { useSettings, useVerifyEmail } from '@deriv/api';
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import WithdrawalVerification from '../WithdrawalVerification';
+import '@testing-library/jest-dom';
+
+jest.mock('../WithdrawalVerificationRequest', () => ({
+    ...jest.requireActual('../WithdrawalVerificationRequest'),
```

Comment:
minor - it seems like there is mock, but also, that mock actually only includes original... looks like leftover from development?

Reply:
unless you actually want to mock those components, which also would be a valid approach

Reply:
This is a workaround which was added in this fix: https://github.com/binary-com/deriv-app/pull/12417/commits/3ab598ce63980a23949207f6ef0224474b6fe38d

As the previous implementation using this method:
`import { WithdrawalVerificationRequest } from '../WithdrawalVerificationRequest';`
would mark the import as unused and returns an eslint error, but we still need the actual content of the component as mocking it would render the components inside i.e. button unusable

Reply:
Updated and removed unneeded mock